app/views/admin/stats/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<div class="stats">
+<h1>Statistics for <%= application_name %>
+<%= Sufia::VERSION %></h1>
+<br />
+<h3>Total Files: <%= @files_count[:total] %> </h3>
```

Comment:
Skipped `h2`. Not a big deal to me, but does that conform with your usability guidelines?